docs/content/guides/dagster/asset-versioning-and-caching.mdx

```diff
@@ -218,25 +237,25 @@ width={2662}
 height={1748}
 />
 
-Notice that `multiplied_number` is no longer stale, despite the fact that we did not materialize it! Here's what happened-- `versioned_number` is no longer stale, so `multiplied_number` is no longer automatically stale. Dagster then compared the data version of `versioned_number` last used to materializate `multiplied_number` to the current data version of `versioned_number`. Since this comparison shows that the data version of `versioned_number` has not changed, Dagster knows that `multiplied_number` is not stale.
+Notice that `multiplied_number` is no longer stale, even though we didn't materialize it! Here's what happened: `versioned_number` is no longer stale, so `multiplied_number` is no longer automatically stale. Dagster then compared the data version of `versioned_number` last used to materialize `multiplied_number` to the current data version of `versioned_number`. Since this comparison shows that the data version of `versioned_number` hasn't changed, Dagster knows that `multiplied_number` isn't stale.
 
-Note that if `versioned_number` had used a Dagster-generated data versions, `multiplied_number` would be stale. The data version of `versioned_number` would have changed due to its updated code version, despite the fact that the returned value did not change. `multiplied_number` would then register as stale based off of the updated data version of `versioned_number`.
+**Note**: If `versioned_number` had used a Dagster-generated data version, `multiplied_number` would be stale. The data version of `versioned_number` would have changed due to its updated code version, despite the fact that the returned value did not change. `multiplied_number` would then register as stale based on the updated data version of `versioned_number`.
 
 ---
 
-## Step Four: Staleness with Source Assets
+## Step four: Staleness with source assets
 
-In the real world, data pipelines depend on external upstream data. So far in this tutorial, we haven't used any external data-- we've been substituting hardcoded data in the asset at the root of our graph, and using a code version as a stand-in for the version of that data. We can do better than this.
+In the real world, data pipelines depend on external upstream data. So far in this guide, we haven't used any external data: we've been substituting hardcoded data in the asset at the root of our graph and using a code version as a stand-in for the version of that data. We can do better than this.
```

Comment:
>haven't used any external data: we've been

IMO ":" doesn't fit here-- were you thinking ";" instead?

Alternatively:

>external data. Instead, we've been....

Reply:
Yes; ty